### PR TITLE
upstream merge but real

### PR DESCRIPTION
**Pull request by deltanedas**
merge commit...

### CODE REVIEW

File: Content.Client/Commands/ActionsCommands.cs [hunk 1]
```diff
@@ -50,7 +46,7 @@ public override void Execute(IConsoleShell shell, string argStr, string[] args)
     {
         if (args.Length != 1)
         {
-            LoadActs(); // DeltaV - Load from a file dialogue instead
```
Comment (IamVelcroboy) [inline Content.Client/Commands/ActionsCommands.cs:53]: I need this back :(

Comment (deltanedas) [reply Content.Client/Commands/ActionsCommands.cs:53]: idk i think it was breaking mapping

Comment (IamVelcroboy) [reply Content.Client/Commands/ActionsCommands.cs:53]: Fwiw, I added it back so I could do some mapping last night w/ no issues. I don't know what will happen if we re-enable the mapping editor though.

Comment (deltanedas) [reply Content.Client/Commands/ActionsCommands.cs:53]: make a pr then, if it breaks map editor again whenever map editor works it can be reverted

**Pull request merged**